README.md

```diff
@@ -60,9 +61,12 @@ echo "done."
 7. `npm start`
 8. Navigate to `https://localhost` in your browser (or just `http` if LINEUPLIST_FORCE_HTTP is set to true)
 
+Note: for Windows, all steps are the same except step 1 which require equivalent installations of redis and npm on the path, and step 8 where the application will start on port 8080 rather than 80 if in non-PROD
```

Comment:
Can you make this match the formatting of the `macOS` instructions please? Kind of petty, but basically just copy / paste the above block and change the heading to Windows. I'd rather have the information duplicated and clear then missing this line

For step one you can change to something like: "Install both redis and npm either through a package manager like Chocolatey or directly from their source installers"

Reply:
Done